CPAC/resources/configs/pipeline_config_benchmark-FNIRT.yml

```diff
@@ -82,11 +82,6 @@ anatomical_preproc:
     # this is a fork option
     using: [BET]
 
-    FSL-BET: 
-
-      # Mask created along with skull stripping. It should be `On`, if selected functionalMasking :  ['Anatomical_Refined'] and `FSL` as skull-stripping method.
-      mask_boolean: Off
-
```

Comment:
Yes, this is my mistake it got pulled into this PR- but it's resolving an "internal" error (isolated to `develop`) so I'll keep it in, instead of creating a new PR for it.